packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -676,6 +676,10 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
   /// Animates the position such that the given object is as visible as possible
   /// by just scrolling this position.
   ///
+  /// The [padding] is used to add extra space around the [object] when revealing it.
+  /// For example, `EdgeInsets.only(bottom: 16.0)` will ensure an additional 16 pixels
+  /// of space are visible below the [object].
+  ///
```

Comment:
This section could do with a `See also` reference back to the FocusNode properties that apply this padding.